docs/admin-panel/hierarchy/ERC721/features.md

```diff
@@ -0,0 +1,64 @@
+
+One practical example of its use is in the sale of mystery boxes. If the NFTs within the mystery boxes are not yet available, the contract can be paused until they are ready. This ensures that the users cannot access the NFTs until the contract is ready, adding an extra layer of security and control to the process. The Pausable feature of ERC721 contracts allows for greater flexibility and control, making it an important component for any project utilizing NFTs.
+
+
+### UPGRADEABLE
+
+The **Upgradeable** feature allows NFTs to be upgraded with attributes such as level, power, stamina, etc. It provides a new level of functionality to NFTs by allowing owners to enhance their assets. This can increase the value and desirability of NFTs, making them more appealing to buyers and investors.
+
+In a gaming platform, for example, players can upgrade their NFTs to improve their in-game abilities or earn more in-game currency. This not only enhances the user's experience but also positively influences the overall economy of the token as upgraded tokens may hold a higher value.
+
+
+### GENES
+
+The **Genes** feature allows for the creation and management of unique, inheritable traits within NFTs. This allows for a new level of complexity and depth in the NFT ecosystem, creating new opportunities for collectibility and gameplay. The GENES feature has been utilized by popular NFT projects such as [Axie Infinity](https://axieinfinity.com/) and [CryptoKitties](https://www.cryptokitties.co/), where players can breed and trade creatures with unique traits. One more example is [Derace](https://derace.com/) that allow for the creation of rare and valuable horses NFTs, where the traits of the horses can be passed down through generations. The addition of the GENES feature opens up new possibilities for NFT creators, collectors, and gamers.
+
+
+### RANDOM
+
+The **RANDOM** feature is an important aspect of the ERC721 contract, as it provides a way to randomly generate events within a blockchain application. To ensure the fairness and transparency of the random number generation process, we use the popular decentralized oracle service, [ChainLink](/docs/integrations/chain-link/).
+
+Combining the RANDOM feature with the UPGRADABLE feature also provides new possibilities for NFT-based gaming experiences. Players can be surprised by the rarity of the NFT they receive and can then work to upgrade it and increase its value. This creates a unique and dynamic NFT economy, where the value of NFTs is determined by both player effort and luck.
+
+You can also combine RANDOM with other features such as MYSTERY BOX, and this can enhance the overall user experience and drive engagement. The use of RANDOM in NFT projects, such as in gaming, can greatly add to the value and desirability of the NFTs, leading to a more active and thriving community and economy.
+
```

Comment:
there is one more feature RENTABLE which indicates the presence of the ERC4907 extension